app/views/posts/index.html.erb

```diff
@@ -1 +1,17 @@
-<h1>Here is a list of posts for a given user index</h1>
\ No newline at end of file
+<div>
+<%  @user = User.find(params[:user_id]) %>
+  <%= render 'share/user', :user => @user %>
+<% @posts = Post.all %>
+<% @comments = Comment.all %>
+<% @users = User.all %>
+<% if !@posts %>
+  <h4 class="card-title">There are no posts from this user yet<h4>
+<% else %>
+<% @posts.each do |p| %>
```

Comment:
- [ ] Please use  `post.recent_comments` to render 5 most recent comments in the `app/views/posts/index.html.erb` page it may look like this :point_down: 

```ruby
 <% post.recent_comments.each do |comment| %>
  <p><%= comment.author.name %>: <%= comment.text %></p> 
 <% end %>
```